.github/actions/build-qt/action.yml

```diff
@@ -0,0 +1,110 @@
+name: 'Build Qt'
+description: 'Builds Qt for obs-deps with specified architecture and build config'
+inputs:
+  target:
+    description: 'Build target for Qt'
+    required: true
+  config:
+    description: 'Build configuration'
+    required: false
+    default: 'Release'
+  qtVersion:
+    description: 'Qt version to build'
+    required: false
+    default: '6'
+  workingDirectory:
+    description: 'Working directory for repository action'
+    required: false
+    default: ${{ github.workspace }}
+runs:
+  using: 'composite'
```

Comment:
Quotes are usually only necessary if a string contains a value that could be misinterpreted as another type or yaml token, is multiline, and to force non-string types to be converted to strings.

I chose to keep them quoted here because it's what we do for other action definition blocks as well. We could clean them all up later.

Reply:
> I chose to keep them quoted here because it's what we do for other action definition blocks as well.

They aren't quoted in `.github/workflows/scheduled.yaml` or newly added sections of `.github/workflows/main.yml`, so I found it odd to be quoting now if we're just going to change it later.

Reply:
That is correct - each PR just represents bits and pieces of an entire overhaul. Once they're all merged (as they are meant to be) the workflow files will all use the unquoted pattern as much as possible

Reply:
This is a brand new file. Why start with quoted strings here if the end-goal is to remove them?

Reply:
Because actions keep them in their declaration sections, while everything else doesn't. It was just easier to effectively copy and paste the existing actions and adjust their script blocks. I can update those in the same way if that's preferred.